python/grass/jupyter/display.py

```diff
@@ -92,7 +92,10 @@ def __init__(
         self._env["GRASS_LEGEND_FILE"] = str(self._legend_file)
 
     def run(self, module, **kwargs):
-        """Run modules from "d." GRASS library"""
+        """Run modules from "d." GRASS library using grass.script.run_command().
+
+        :param str module: name of GRASS module
+        :param **kwargs: named arguments passed to run_command()"""
```

Comment:
Please check Sphinx for both the kwargs syntax and reffering to functions (I'm not sure if the `**` is needed and cursive may be recommend instead of `()` when referring to functions).